README.rst

```diff
@@ -42,6 +42,18 @@ For general help content, pass in the ``-h`` parameter:
 
     $ mssql-scripter -h
 
+
+Docker
+-----
+
+        docker pull felipegouveiae/mssql-scripter:latest
```

Comment:
Unfortunately, we can't have an external dockerfile like this in our docs. So please remove this - and then I'll see about getting the below dockerfile published under the Microsoft account.